model/src/main/proto/onboarding.proto

```diff
@@ -24,6 +24,15 @@ message AppStartupState {
     // continue using it. Instead, they should be shown a prompt suggesting that they update the app
     // via the Play Store.
     APP_IS_DEPRECATED = 3;
```

Comment:
Perharps `FORCED_UPDATE_AVAILABLE` to correspond to `OPTIONAL_UPDATE_AVAILABLE` below?

Reply:
I meant APP_IS_DEPRECATED means the same as should force update, hence could be renamed. However feel free to ignore this suggestion.

Reply:
That's what I am using it for. APP_IS_DEPRECATED will show that the user is required to update the app.